airflow/models/xcom.py

```diff
@@ -62,13 +62,7 @@ def init_on_load(self):
         Called by the ORM after the instance has been loaded from the DB or otherwise reconstituted
         i.e automatically deserialize Xcom value when loading from DB.
         """
-        try:
-            self.value = self.orm_deserialize_value()
-        except (UnicodeEncodeError, ValueError):
```

Comment:
Those exceptions came from Python2 `json.loads`, have been replaced by `JSONDecodeError` (_as below_) in Python3.